.github/workflows/_test_build_nix_fake_prerelease.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Test Build Nix fake prerelease"
+
+on:
+  workflow_call:
+    inputs:
+      original_repo_name:
+        required: true
+        type: string
+      repo_full_name:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+      integration:
+        required: true
+        type: string
+    secrets:
+      docker_hub_id:
+        description: "docker hub id"
+        required: true
+      docker_hub_password:
+        description: "docker hub password"
+        required: true
+      gpg_passphrase:
+        description: "GPG passphrase"
+        required: true
+      gpg_private_key_base64:
+        description: "GPG private key base64 encoded"
+        required: true
+
+jobs:
+  test-build-nix:
+    name: Test binary compilation for all platforms:arch
+    runs-on: ubuntu-20.04
+    env:
+      INTEGRATION: ${{ inputs.integration }}
+      ORIGINAL_REPO_NAME: ${{ inputs.original_repo_name }}
+      REPO_FULL_NAME: ${{ inputs.repo_full_name }}
```

Comment:
this looks also like legacy thing used for go path before moving to go modules

Reply:
yes i checked and ORIGINAL_REPO_NAME looks like is used as a path on windows which I think is not needed, and the REPO_FULL_NAME is only used in pre-release

Reply:
In christian PR this was used instead of having an input more:
```
repo_name: ${{ github.event.repository.full_name }}
```

Reply:
Maybe with "custom workflows" we needed to pass it as input (otherwise the action would have taken its own repo name), but with the reusable ones, I think as guillermo, we should remove those if possible

https://github.com/newrelic/coreint-automation/pull/42/files

Reply:
Updated in both workflows 👍